contrib/scripts/release.sh

```diff
@@ -6,6 +6,7 @@ args=(
     "-p nostr"
     "-p nostr-database"
     "-p nostr-lmdb"
+    "-p nostr-mls-memory-storage"
```

Comment:
This should be put under the `nostr-mls-storage`, otherwise the release will fail since `nostr-mls-memory-storage` depends on it.